### ISSUE TITLE
Allow passing arguments to model calls

### ISSUE THREAD

**Issue by mys007** [keisen/tf-keras-vis#26]
Currently, the codebase does not allow to pass arguments to model calls, e.g.

https://github.com/keisen/tf-keras-vis/blob/fdfb61b325fe6f49e7297fc38f37e9f02bfd918f/tf_keras_vis/gradcam.py#L58

This means one cannot influence whether a model is called in inference or training mode, as e.g. data augmentation layers are on by default (https://github.com/tensorflow/tensorflow/blob/v2.3.0/tensorflow/python/keras/layers/preprocessing/image_preprocessing.py#L396) and global setting of learning phase is deprecated (https://www.tensorflow.org/api_docs/python/tf/keras/backend/set_learning_phase). I suggest the training mode be passed as a parameter of e.g. Gradcam.__call__.

**Comment by keisen** [comment]
Thank you for your reporting!
I will address it immediately.

**Comment by mys007** [comment]
Awesome, really great maintenance! Thanks.

**Comment by keisen** [comment]
You're welcome!
BTW, if you would like, please click the Github Stars of this repository.
